msvc/libgc.targets

```diff
@@ -0,0 +1,181 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="libmonogc_la_sources">
+    <ClCompile Include="$(MonoSourceLocation)\libgc\allchblk.c">
+		  <ExcludedFromBuild Condition="'$(MONO_TARGET_GC)'!='boehm'">true</ExcludedFromBuild>
+	  </ClCompile>
```

Comment:
there's some weird mix of tabs/spaces here and below.

Reply:
I will make a pass over the files and remove some of the editor artifacts.